packages/lodestar-validator/src/api/impl/rest/events/events.ts

```diff
@@ -30,4 +31,21 @@ export class RestEventsApi implements IEventsApi {
       };
     });
   }
+
+  public getEventEmitter(topics: BeaconEventType[], signal: AbortSignal): EventSourceBeaconEventEmitter {
```

Comment:
Why not on start use getEventStream and emit events from there instead of duplicate implementation with different interface?

Reply:
I guess because this implementation is a lot simpler this way? Not having to deal with async functions or generators inside the EventEmitter implementation.

Reply:
Hmm but you can implement this without using EventSource directly. 
Basically:
```typescript
getEventEmmitter(topics) {

   for await (const event in this.getEventStream(topics)) {
      something.emit(event.topic, event.message);
   }

}
```
And if you put it as utility function somewhere, you don't have to reimplement on lodestar side